actions/lib/formatters.py

```diff
@@ -250,6 +251,18 @@ def repo_to_dict(repo):
 
     return result
 
+def org_to_dict(org):
+    result = {}
+
+    result['name'] = org.name
+    result['id'] = org.id
+    result['login'] = org.login
+    result['updated_at'] = org.updated_at.isoformat()
+    result['html_url'] = org.html_url
```

Comment:
Can we also include the `url` (i.e. the api one)?  Have a feeling may need it also

Reply:
OK.  I added it.